system/Entity/Exceptions/CastException.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * This file is part of the CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Entity\Exceptions;
+
+use CodeIgniter\Exceptions\FrameworkException;
+
+class CastException extends FrameworkException
+{
+	public static function forInvalidInterface($class)
```

Comment:
For our API documentation, please add docblocks on each method. Then add a docblock at the class level.